### ISSUE TITLE
YAxis size calculations

### ISSUE THREAD

**Issue by AlBirdie** [danielgindi/ios-charts#480]
I've noticed that the current master branch resizes the y axis when the axis labels change, for instance during scrolling. 
That means, that when you manipulate the current x axis range, it can happen that the max width of the y labels change, resulting in a resize of the chart drawing panel.
Can be disable this feature and go back to truncating the y axis labels instead?
I couldn't find a property for that.

**Comment by danielgindi** [comment]
It was actually never truncated - just did not calculate when scrolling... This has changed many commits ago!
You can set a fixed width for the Y axis to disable the constant updates

**Comment by AlBirdie** [comment]
Oh I see.
Didn't keep up the changes lately as I had to implement the Android version of the chart.
Ok then, fixed size it is. Thanks for your help, Daniel!

**Comment by danielgindi** [comment]
NP :-)

**Comment by AlBirdie** [comment]
@danielgindi, is there a way to actually truncate the y-axis labels? Turns out that constantly re-adjusting my synchronised chart viewports (I've got several charts stacked vertically that need to have the same y-axis width), looks kinda wrong during manipulation of the chart's viewport. Instead, setting the y-axes widths to the maximum size after the initial draw and then leave it that way looks way better.
However, in order to not have the y-axis labels reach into the chart's viewport, I need them to be truncated.
This should be doable in the ChartYAxisRenderer (if _yAxis.maxWidth > 0...), and using a lineBreakMode, right?
This is probably a feature many require, so I'd like to give this a go in the foreseeable future.
